README.md

```diff
@@ -480,6 +468,109 @@ This command and related code in SwiftLint is subject to substantial changes at
 any time while this feature is marked as experimental. Analyzer rules also tend
 to be considerably slower than lint rules.
 
+## Using Multiple Configuration Files
+
+SwiftLint offers a variety of ways to include multiple configuration files.
+Multiple configuration files get merged into one single configuration that is then applied
+just as a single configuration file would get applied.
+
+There are quite a lot of use cases where using multiple configuration files could be helpful:
+
+For instance, one could use a team-wide shared SwiftLint configuration while allowing overrrides
+in each project via a child configuration file.
+
+Team-Wide Configuration:
+
+```yaml
+disabled_rules:
+- force_cast
+```
+
+Project-Specific Configuration :
+
+```yaml
+opt_in_rules:
+- force_cast
+```
+
+### Child / Parent Configs (Locally)
+
+You can specify a `child_config`  and / or a `parent_config` reference within a configuration file.
+These references should be local paths relative to the folder of the configuration file they are specified in.
+This even works recursively, as long as there are no cycles and no ambiguities.
+
+**A child config is treated as a refinement and therefore has a higher priority**,
+while a parent config is considered a base with lower priority in case of conflicts.
+
+Here's an example, assuming you have the following file structure:
+
+```
+ProjectRoot
+    |_ .swiftlint.yml
+    |_ .swiftlint_refinement.yml
+    |_ Base
+        |_ .swiftlint_base.yml
+```
+
+To include both the refinement and the base file, your `.swiftlint.yml` should look like this:
+
+```yaml
+child_config: .swiftlint_refinement.yml
+parent_config: Base/.swiftlint_base.yml
+```
+
+When merging parent and child configs, `included` and `excluded` configurations
+are processed carefully to account for differences in root paths.
+
+### Child / Parent Configs (Remotely)
+
+Just as you can provide local `child_config` / `parent_config` references, instead of
+referencing local paths, you can just put urls that lead to configuration files.
+In order for SwiftLint to detect these remote references, they must start with `http://` or `https://`.
+
+The referenced remote configuration files may even recursively reference other
+remote configuration files, but aren't allowed to include local references.
+
+Using a remote reference, your `.swiftlint.yml` could look like this:
+
+```yaml
+parent_config: https://myteamserver.com/our-base-swiftlint-config.yml
+```
+
+Every time you run SwiftLint and have an internet connection, SwiftLint tries to get a new version of
```

Comment:
Just to make sure, if there are no config references starting with `http(s)://`, no network calls are made, right?

```suggestion
Every time you run SwiftLint and have an Internet connection, SwiftLint tries to get a new version of
```

Reply:
Nope, no network calls are made ;)